modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -21,6 +21,7 @@
 import io.swagger.v3.oas.models.OpenAPI;
 import io.swagger.v3.oas.models.media.Schema;
 import io.swagger.v3.parser.util.SchemaTypeUtil;
+import org.mozilla.javascript.optimizer.Codegen;
```

Comment:
What is this for?

Reply:
Nothing. Removed it. Was added by accident and I didn't catch it